app/celery/provider_tasks.py

```diff
@@ -91,6 +91,13 @@ def check_sms_delivery_receipt(self, message_id, notification_id, sent_at):
     bind=True, name="deliver_sms", max_retries=48, default_retry_delay=300
 )
 def deliver_sms(self, notification_id):
+    """
+    This logic will branch off to the final step in delivering
+    the notification to sns.
```

Comment:
Not to be pedantic, but can these be conformed to PEP-257?

The first line is a complete sentence (not broken up over multiple lines). There needs to be a separate blank line after the first line. It also needs to be an active, direct statement, not merely a description of what it does.

`Branch off to the final step in delivering the notification to sns.` would be a more compliant docstring first line.

Reply:
@xlorepdarkhelm Oof, it looks like I am breaking some conventions here. Thank you for pointing me to the right place with PEP-257. I'll get these sorted out!